lib/bambutag/BambuTagParser.cpp

```
@@ -0,0 +1,65 @@
+#include "BambuTagParser.h"
+#include <cstring>
+
+static uint16_t readU16LE(const uint8_t* p) {
+    return (uint16_t)p[0] | ((uint16_t)p[1] << 8);
+}
+
+static float readFloatLE(const uint8_t* p) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)